test_xdb/.sqlfluff

```diff
@@ -0,0 +1,28 @@
+[sqlfluff]
+dialect = postgres
+exclude_rules = L009,L016,L031,L034
+
+# L009 - newline at end of file - dbt seems to remove those trailing newlines
+# L016 - line too long - compiled lines end up longer than templated lines
+# L031 - Avoid table aliases - aliases in general are ok
+# L034 - wildcards, simple, calc field ordering - currently too many reorderings to satsify
```

Comment:
Might be ok removing L034 from this list since it's there probably aren't too many changes necessary to satisfy it.